irods/manager/data_object_manager.py

```diff
@@ -20,6 +20,24 @@
 import logging
 
 
+# Utility method to call __del__ if it exists anywhere in superclasses' MRO
+
+def call___del__if_exists(super_):
+    next_finalizer_in_MRO = getattr(super_,'__del__',None)
+    if next_finalizer_in_MRO:
+        next_finalizer_in_MRO()
+
+class MyBufferedRandom(io.BufferedRandom):
```

Comment:
Consider replacing `My` and/or renaming this to be more descriptive.

For example _(you don't have to use these)_:
- `AutoCloseableBufferedRandom`
- `irods.io.BufferedRandom` _(this exists in the iRODS module)_
- `ManagedBufferedRandom`

Reply:
agree, will choose one of those....